google/resource_dns_record_set.go

```diff
@@ -55,6 +57,101 @@ func resourceDnsRecordSet() *schema.Resource {
 	}
 }
 
+// This catches most invalid DNS names, though it might not catch some invlaid non-ASCII names
+// it does not implement the TOASCII functionality defined in https://tools.ietf.org/html/rfc3454
+// See:
+// https://tools.ietf.org/html/rfc1034 DOMAIN NAMES - CONCEPTS AND FACILITIES
+// https://tools.ietf.org/html/rfc3490 Internationalizing Domain Names in Applications (IDNA)
+// https://tools.ietf.org/html/rfc3454 Preparation of Internationalized Strings ('stringprep')
+func validate_dns_rrset_name(name string) error {
```

Comment:
golang style is to use mixedCaps for pretty much everything: https://golang.org/doc/effective_go.html#mixed-caps

Reply:
Will change

Reply:
Done.